pubnub/enums.py

```diff
@@ -19,7 +19,7 @@ def string(cls, method):
             return "PATCH"
 
 
-class PNStatusCategory(object):
+class PNStatusCategory(Enum):
```

Comment:
Disclaimer: For most of situations this doesn't change anything. `Enum` inherits from `object` (which is leftover from py2.7 btw) and only adds functionality like this `status.category.name` to extract the name based on the value